Jenkinsfile

```diff
@@ -49,34 +58,53 @@ node('lisk-explorer-01'){
         grunt candles:build
         '''
       } catch (err) {
-        currentBuild.result = 'FAILURE'
-        error('Stopping build, candles build failed')
+        echo "Error: ${err}"
+        fail('Stopping build, candles build failed')
       }
     }
 
     stage ('Start Lisk ') {
       try {
-        sh '''#!/bin/bash
+
+        sh '''
+        N=${EXECUTOR_NUMBER:-0}
+        cd ~/lisk-Linux-x86_64
+        # work around core bug: config.json gets overwritten; use backup
+        cp .config.json config_$N.json
+        # change core port
+        sed -i -r -e "s/^(.*ort\\":) 4000,/\\1 400$N,/" config_$N.json
+        # disable redis
+        sed -i -r -e "s/^(\\s*\\"cacheEnabled\\":) true/\\1 false/" config_$N.json
+        # change postgres databse
+        sed -i -r -e "s/^(\\s*\\"database\\": \\"lisk_test)\\",/\\1_$N\\",/" config_$N.json
+        cp etc/pm2-lisk.json etc/pm2-lisk_$N.json
+        sed -i -r -e "s/config.json/config_$N.json/" etc/pm2-lisk_$N.json
+        sed -i -r -e "s/(lisk.app)/\\1_$N/" etc/pm2-lisk_$N.json
```

Comment:
my recommendation is to keep a preconfigured config.json in the test folder of explorer that can be copied into place, removing this messy logic.